variables.tf

```diff
@@ -193,3 +193,9 @@ variable "s3_bucket_force_destroy" {
   description = "A boolean that indicates all objects should be deleted from the CodePipeline artifact store S3 bucket so that the bucket can be destroyed without error"
   default     = false
 }
+
+variable "cache_type" {
+  type        = string
+  default     = "S3"
+  description = "The type of storage that will be used for the AWS CodeBuild project cache. Valid values: NO_CACHE, LOCAL, and S3.  Defaults to NO_CACHE.  If cache_type is S3, it will create an S3 bucket for storing codebuild cache inside"
```

Comment:
@jamengual Your description say "Defaults to NO_CACHE.", but the default in the variable is "S3". Wanna update it to "NO_CACHE"?

Reply:
fixed